server/routes/coordinator.js

```diff
@@ -84,7 +87,7 @@ module.exports = function Index({ reportService, involvedStaffService, reviewSer
         await reportService.deleteReport(res.locals.user.username, reportId)
```

Comment:
so if you load the report before this line, (maybe line 85? as you will always need it whether confirmed or not)  you should be able to grab the status of the report and determine the correct location at this point only.

Reply:
Yes, that could work